third_party/mkdocs-material/README.md

```diff
@@ -0,0 +1,22 @@
+# Material for MkDocs
+
+This directory includes files forked from https://github.com/squidfunk/mkdocs-material.
+
+See:
+
+* https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme
+* https://github.com/squidfunk/mkdocs-material/tree/master/src/partials
+
+These override files can either be included in `mkdocs.yml` either via
+
+```yaml
+  custom_dir: overrides
+  # Then symlink to files from overrides/partials/... to this directory
+```
+
+or
+
+```yaml
+  custom_dir: ../../third_party/mkdocs-material/overrides
+  # Then have no override files outside of this third_party/ directory
+```
```

Comment:
I would just say the one that we're doing (symlinking). I'd also mention that they're in third_party and we're doing the symlinking to make the third-party code distinction clear